engines/bops_submissions/app/services/bops_submissions/creation_service.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module BopsSubmissions
+  class CreationService
+    HEADERS = %w[
+      User-Agent
+      Content-Type
+      Accept
+      X-Request-Id
+      X-Forwarded-For
+      X-Real-IP
+      Host
+      Referer
+    ].freeze
+
+    def initialize(params:, headers:, local_authority:)
+      @params = params
+      @headers = headers
+      @local_authority = local_authority
+    end
+
+    attr_reader :params, :headers, :local_authority
+
+    def call
+      submission = local_authority.submissions.create!(
+        request_headers: filtered_request_headers,
+        request_body: permitted_request_params
+      )
+
+      submission.update!(external_uuid: SecureRandom.uuid_v7)
```

Comment:
What is this UUID for?

Reply:
Agreed with PP that they will receive this as a success message and possible identifier for future interaction with that resource

Reply:
Also don't want to be giving back a database-generated ID

Reply:
Agreed we shouldn't be sending back the database-generated integer primary keys.

Reply:
Hm I guess we could make it the primary key